<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ARTWindow</class>
 <widget class="QWidget" name="ARTWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARTWindow</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>931</width>
     <height>302</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1" colspan="2">
     <widget class="QLabel" name="lSampleRate">
      <property name="text">
       <string>采样率</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="tbAmplitude">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="tbSampleRate">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLabel" name="lSamplePerChan">
      <property name="text">
       <string>周期采样点</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="tbSamplePerChan">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLabel" name="lAmplitude">
      <property name="text">
       <string>调制深度</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QPushButton" name="bnCloseDevice">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>关闭设备</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLineEdit" name="tbTerm">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="4" colspan="2">
     <widget class="QPushButton" name="bnGetParam">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>获取参数</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="2">
     <widget class="QPushButton" name="bnSetParam">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>配置参数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QPushButton" name="bnOpenDevice">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>打开采集卡</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="2">
     <widget class="QPushButton" name="bnStartAO">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>开始输出</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLabel" name="lFrequency">
      <property name="text">
       <string>调制频率</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLineEdit" name="tbFrequency">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLabel" name="lTerm">
      <property name="text">
       <string>周期</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="10">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QCheckBox" name="cbOneSample">
      <property name="text">
       <string>单点输出模式</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" rowspan="2" colspan="2">
     <widget class="QLabel" name="lBiasVoltage">
      <property name="text">
       <string>偏置电压</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3" rowspan="2">
     <widget class="QLineEdit" name="tbBiasVoltage">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="4" rowspan="2" colspan="2">
     <widget class="QPushButton" name="bnStopAO">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>停止输出</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2" colspan="4">
     <widget class="QLabel" name="lFormula">
      <property name="text">
       <string>调制周期 = 周期采样点 / 采样率</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
